application/tests/cheatsheets_workstream_f_test.py

```
@@ -288,5 +294,95 @@ def test_category_tag_is_stripped(self) -> None:
 
+def _run_cli(argv):
+    """Invoke wf.main(argv), capturing (exit_code, stdout, stderr)."""
+    out, err = io.StringIO(), io.StringIO()
+    with contextlib.redirect_stdout(out), contextlib.redirect_stderr(err):
+        code = wf.main(argv)
+    return code, out.getvalue(), err.getvalue()
+
+
+class TestCliValidate(unittest.TestCase):
+    def test_valid_fixture_exits_zero(self) -> None:
+        code, out, err = _run_cli(["validate", VALID_FIXTURE])
```

Ruff (0.16.1)

[warning] Unpacked variable `err` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)